clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -893,7 +796,7 @@ def create_adni_scans_files(conversion_path, bids_subjs_paths):
                             f"{Fore.RED}No information found for file {file_name}.{Fore.RESET}"
                         )
 
-                    scans_df = scans_df.fillna("n/a")
+                    scans_df = scans_df.fillna("")
```

Comment:
Why the empty string instead of "n/a" previously? May be worth adding a comment.